scripts/build.sh

```diff
@@ -1,25 +1,26 @@
 #!/bin/bash
 # Build require-ale and minified assets for distribution.
+(set -o igncr) 2>/dev/null && set -o igncr; # For Cygwin on Windows compatibility
 set -o errexit
-./scripts/make-mini-d3.sh # TODO: remove
+sh scripts/make-mini-d3.sh # TODO: remove
 
 # Transpile individual files. This is useful if another module,
 # e.g. cycledash, wants to require('pileup').
 # The dist/test files are required for code coverage
 mkdir -p dist/test/{data,source,viz}
-babel src --retain-lines --ignore src/lib --out-dir dist
+node_modules/.bin/babel src --retain-lines --ignore src/lib --out-dir dist
```

Comment:
ah, gotcha! Didn't realize that the `npm run build` was failing due to the way `scripts` are defined in `package.json`.

I tried googling a solution for this, realized that this is a common issue, and it is NPM-related rather than pileup itself. Furthermore, NPM people officially do **not** support `cygwin` and recommend against using it: https://github.com/npm/npm#installing-on-cygwin

Although it would be great to support cygwin, I don't think we should make strong assumptions about people's setup and just do the things in the NPM way as much as possible. I didn't know about the `cygwin/npm` interaction was that problematic and I think it is not up to us to start adjusting things for a platform that is not supported. 

I really appreciate the PR and your contribution, but I don't think we should merge this into pileup.